lib/util.js

```diff
@@ -167,6 +167,18 @@ allProfiles.forEach(i => {
 });
 exports.allProfiles = allProfiles;
 
+[...TRProfiles.map(x => `TR/${x}`), ...filesSUBM.map(x => `SUBM/${x}`)].forEach(
+    i => {
+        const valid =
+            /((TR|SUBM)\/([A-Za-z]+\/)?([A-Z][A-Z-]*[A-Z](-Echidna)?))\.js$/.exec(
+                i
+            );
+        if (valid && valid.length > 3)
+            // eslint-disable-next-line import/no-dynamic-require
+            profiles[valid[4]] = require(`./profiles/${valid[1]}`);
+    }
+);
+
```

Comment:
What's the purpose of that change? It seems to be duplicating the previous lines.